api/proto/banyandb/model/v1/common.proto

```diff
@@ -77,3 +77,17 @@ enum AggregationFunction {
   AGGREGATION_FUNCTION_COUNT = 4;
   AGGREGATION_FUNCTION_SUM = 5;
 }
+
+enum MeasureAggregate {
+  MEASURE_AGGREGATE_UNSPECIFIED = 0;
+  MEASURE_AGGREGATE_MIN = 1;
+  MEASURE_AGGREGATE_MAX = 2;
+  MEASURE_AGGREGATE_COUNT = 3;
+  MEASURE_AGGREGATE_SUM = 4;
+  MEASURE_AGGREGATE_AVG = 5;
+  MEASURE_AGGREGATE_PERCENT = 6;
+  MEASURE_AGGREGATE_RATE = 7;
+  MEASURE_AGGREGATE_HISTOGRAM = 8;
+  MEASURE_AGGREGATE_PERCENTILE2 = 9;
```

Comment:
@wu-sheng Would you mind if we call it 'percentile2', which aligns with the name defined in OAP?

Reply:
I think it is OK to align. Do you have percentile? I think we need that too.

Reply:
Ok, I'll add that.